pkg/controller/volume/attachdetach/cache/actual_state_of_world_test.go

```diff
@@ -26,6 +26,36 @@ import (
 	volumetesting "k8s.io/kubernetes/pkg/volume/testing"
 )
 
+func verifyAttachedVolume(
```

Comment:
Before it was in the middle of the code. If I move it to the bottom, it will still has the same effect, I think